### PR TITLE
docs: Ceph disaster recovery

### PR DESCRIPTION
**Pull request by MG40**
The paranthesis around the arguments have been removed to pass the value
Eg: ROOK_CEPH_MON_INITIAL_MEMBERS

Signed-off-by: MG <gmadhavi@gmail.com>

**Description of your changes:**

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/master/development-flow.html#commit-structure).
- [ ] **Skip Tests for Docs**: Add the flag for skipping the build if this is only a documentation change. See [here](https://github.com/rook/rook/blob/master/INSTALL.md#skip-ci) for the flag.
- [ ] **Skip Unrelated Tests**: Add a flag to run tests for a specific storage provider. See [test options](https://github.com/rook/rook/blob/master/INSTALL.md#test-storage-provider).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/master/development-flow.html#submitting-a-pull-request)
- [x] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.

[skip ci]

### CODE REVIEW

**Comment by MG40** [comment]
Removed the paranthesis around the variables.

File: Documentation/ceph-disaster-recovery.md [hunk 1]
```diff
@@ -81,18 +81,18 @@ ceph-mon \
     --log-stderr-prefix=debug \
     --default-log-to-file=false \
     --default-mon-cluster-log-to-file=false \
-    --mon-host=$(ROOK_CEPH_MON_HOST) \
-    --mon-initial-members=$(ROOK_CEPH_MON_INITIAL_MEMBERS) \
+    --mon-host=$ROOK_CEPH_MON_HOST \
+    --mon-initial-members=$ROOK_CEPH_MON_INITIAL_MEMBERS \
```
Comment (leseb) [inline Documentation/ceph-disaster-recovery.md:85]: Where are those variables populated?

Comment (MG40) [reply Documentation/ceph-disaster-recovery.md:85]: I encountered this while executing ceph-mon command in the mon-b pod. 
kubectl -n rook-ceph exec -it <mon-pod> bash
--
[root@rook-ceph-mon-b-5b5dc6ff95-s4789 /]# echo $(ROOK_CEPH_MON_INITIAL_MEMBERS)
bash: ROOK_CEPH_MON_INITIAL_MEMBERS: command not found

[root@rook-ceph-mon-b-5b5dc6ff95-s4789 /]# echo $ROOK_CEPH_MON_INITIAL_MEMBERS 
a,b

Comment (leseb) [reply Documentation/ceph-disaster-recovery.md:85]: Ok, the pod is running, never mind then.

**Review by leseb — APPROVED** [review]

File: Documentation/ceph-disaster-recovery.md [hunk 2]
```diff
@@ -81,18 +81,18 @@ ceph-mon \
...
     --setgroup=ceph \
     --foreground \
     --public-addr=10.100.13.242 \
     --setuser-match-path=/var/lib/ceph/mon/ceph-b/store.db \
-    --public-bind-addr=$(ROOK_POD_IP)
+    --public-bind-addr=$ROOK_POD_IP
 ```
 
-(be sure to remove the single quotes around the `--log-stderr-prefix` flag)
+(be sure to remove the single quotes around the `--log-stderr-prefix` flag and the paranthesis around the variables being passed ROOK_CEPH_MON_HOST, ROOK_CEPH_MON_INITIAL_MEMBERS and ROOK_POD_IP )
```
Comment (travisn) [inline Documentation/ceph-disaster-recovery.md:95]: Is this line still needed? Looks like this PR already removes the parens.

**Review by travisn — CHANGES_REQUESTED** [review]

File: Documentation/ceph-disaster-recovery.md (hunk 2 above)
Comment (MG40) [reply Documentation/ceph-disaster-recovery.md:95]: In the yaml generated - the '' and () appears, hence I thought of extending the instruction to remove the parenthesis.

Comment (travisn) [reply Documentation/ceph-disaster-recovery.md:95]: Ok, I see now where you're referring to in the yaml that comes from the pod spec, thanks

**Review by travisn — APPROVED** [review]

**Comment by MG40** [comment]
Thank you.

**Pull request merged**